website/docs/components/table/advanced-table/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,243 @@
+
+## Placement
+
+!!! Info
+
+The column placement property is only relevant within Figma and doesn’t exist as a property in code.
+!!!
+
+Column placement determines the visual styling based on where the column is placed relative to other columns in the Advanced Table.
+
+![Advanced Table column placement example](/assets/components/table/advanced-table/table-col-placement.png)
+
+## Alignment
+
+The alignment of text and content within an Advanced Table impacts the readability and speed at which users can effectively parse the information. The chosen alignment method depends on the content within the cell, and relative position within the advanced table.
+
+!!! Info 
+Helios uses left and right for alignment to match the Ember component API. This documentation follows that convention for consistency across design and development.
+!!!
+
+!!! Do
+
+![Advanced Table column placement example](/assets/components/table/advanced-table/table-alignment-do.png)
+!!!
+
+!!! Dont
+
+![Advanced Table column placement example](/assets/components/table/advanced-table/table-alignment-dont.png)
+!!!
+
+### Start alignment
+
+Align content to the start of the cell by default. This ensures readability across different content types, consistency in content of varying lengths, and alignment between the column header label and the content within the cell.
+- Use start alignment for:
+String and text-based content (unique identifiers or IDs, names and naming conventions, etc).
+- Numerical values that do not contain decimals or floating point numbers.
+- Numerical values that contain periods or other delimiter characters (IP addresses).
+Nested components that display a string or text value, e.g., a [Badge](/components/badge).
+
+![Start alignment of content within an Advanced Table](/assets/components/table/advanced-table/start-alignment-example.png)
+
+### End alignment 
+
+End alignment can be used when expressing numerical values with decimals as this aligns the decimal places vertically.
+Common examples of end alignment include:
+- Financial information and currency amounts.
+- Fractional and floating point values represented with decimals.
+
+![End alignment of content within a table](/assets/components/table/advanced-table/end-alignment-example.png)
+
+End alignment can also be used in the last column of an advanced table to:
+- Highlight a "more options" function pertaining to the content within a row.
+- As a means to visually "bookend" the row with content that is of a similar length, e.g., timestamps, TTL (time-to-live) values, dates.
+
+![End alignment example within a table with a date and more options](/assets/components/table/advanced-table/end-alignment-example-02.png)
+
+!!! Dont
+
+Don’t end align content that is variable in length. This can make the content more difficult to read by forcing an unnatural [reading pattern](/patterns/button-organization?tab=research#layout-and-reading-patterns).
```

Comment:
```suggestion
Don’t right align content that is variable in length. This can make the content more difficult to read by forcing an unnatural [reading pattern](/patterns/button-organization?tab=research#layout-and-reading-patterns).
```